tools/ci_build/build.py

```diff
@@ -1516,11 +1516,16 @@ def run_onnxruntime_tests(args, source_dir, ctest_path, build_dir, configs):
                     run_subprocess([sys.executable, '-m', 'unittest', 'discover', '-s', 'quantization'],
                                    cwd=cwd, dll_path=dll_path)
                     if args.enable_transformers_tool_test:
-                        required = {
-                            'numpy==1.19.2', 'coloredlogs==15.0', 'tf2onnx==1.8.5', 'transformers==4.6.1',
-                            'torch==1.8.1', 'tensorflow==2.5.0', 'onnxconverter-common==1.8.1', 'psutil'}
-                        run_subprocess([sys.executable, '-m', 'pip', 'install', *required])
+                        import numpy
+                        numpy_init_version = numpy.__version__
+                        run_subprocess([sys.executable, '-m', 'pip', 'install', '-r', 'requirements.txt'],
```

Comment:
@wangyems @snnn why do we need pytorch here? it can be torch==1.8.1, torch==1.8.1+cu102, torch==1.8.1+cu111 depending on the environment

Reply:
@wangyems , why do you need pytorch and tensorflow?

Reply:
Just as additional context, if torch==1.8.1 was installed, when the changes on this PR tries to uninstall it, it would fail with permission denied. Probably because pytorch was installed as root and setup.py is executed as regular user.

I believe torch==1.81 shouldnt be included in this requirements.tt, especially at the top level requirements

Reply:
What if we create a venv to for this special test?

Reply:
My concern is changing setup.py, which is dedicated to building ort, to install test dependencies. This is a dangerous practice

I had to remove torch==1.8.1 from requirements.txt to have https://github.com/microsoft/onnxruntime/pull/8027/files passing build

Reply:
But tools/ci_build/requirements.txt was not used in setup.py. This PR didn't touch setup.py